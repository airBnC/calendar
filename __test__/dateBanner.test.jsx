import React, { Component } from 'react';
import DateBanner from '../client/src/components/dateBanner.jsx';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
const currentData = {
	month: 'July',
	year: 18
}
describe('CalendarMonth', () => {
  it('should render correctly', () => {
    const output = shallow(
      <DateBanner currentData={currentData} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
 
});