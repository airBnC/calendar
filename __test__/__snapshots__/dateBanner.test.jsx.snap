// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarMonth should render correctly 1`] = `
<styled.div>
  July
   20
  18
</styled.div>
`;
